packages/zowe-explorer/src/job/ZoweJobNode.ts

```diff
@@ -79,16 +79,17 @@ export class Job extends ZoweTreeNode implements IZoweJobTreeNode {
 
         if (session) {
             this._owner = "*";
-            if (session.ISession.user) {
-                this._owner = session.ISession.user;
+            if (session.ISession?.user) {
+                this._owner = session.ISession?.user;
```

Comment:
I agree on the first question, the safe navigation operator is redundant inside the `if` clause. Will resolve this shortly.

As for the second point, I agree that `ISession.user` should be set, but since the `user` property is optional, it can technically be undefined in some cases (e.g. unit tests that don't properly fill in the values for `ISession`). If we want to avoid using safe navigation here, we should treat the `user` property as required, but that's likely a breaking change.